pcsx2/GS/GSState.cpp

```diff
@@ -1981,6 +1983,8 @@ void GSState::Move()
 	r.top = m_env.TRXPOS.DSAY;
 	r.right = r.left + m_env.TRXREG.RRW;
 	r.bottom = r.top + m_env.TRXREG.RRH;
+
+	s_last_transfer_draw_n = s_n;
```

Comment:
Do you think it's worth adding this to `GSRendererHW::Move()` as well?

Or better yet, whereever the call site for Move() is. Because if it gets HW moved, it won't be updated here (not sure if that's what you intended).

Reply:
I did :/

Reply:
Yeah, fair point, I only noticed after I submitted that the transfer list doesn't get updated on hw move either.

Reply:
yea, we don't need to preload with hw moves, so it would be weird to add it to the list :)